cuml/coordinate_descent_demo.ipynb

```diff
@@ -162,20 +145,14 @@
     "predict_sk = ols_sk.predict(X_test)"
    ]
   },
-  {
-   "cell_type": "markdown",
-   "metadata": {},
-   "source": [
-    "#### Evaluate"
-   ]
-  },
   {
    "cell_type": "code",
    "execution_count": null,
    "metadata": {},
    "outputs": [],
    "source": [
-    "error_sk = mean_squared_error(y_test, predict_sk)"
+    "%%time\n",
+    "r2_score_sk = r2_score(y_cudf_test, predict_sk)"
```

Comment:
Hi @JohnZed,

In that part of code, I am using cuML's r2_score method. y_test and y_cudf_test contains the same data, in different format (pandas DataFrame and cudf DataFrame).

I could:
- use scikit-learn r2_score method: In that case, I should rename the imports to r2_score_sk and r2_score_cu
- use cuML r2_score method, and use y_cudf_test, because both y_test and y_cudf_test contain the same data. This is the current approach.
- use cuML r2_score method, and convert y_test to cudf.Series, which would be the same as what it is being done at the moment.

Please, let me know which is your preferred option, and I will change the notebooks accordingly.

Thanks!
Miguel

Reply:
I looked over the other similar notebooks, and many of them only use one r2_score method (rather than separate methods as in option 1). Most use sklearn's as we previously didn't have a cuml one. However, I think using just the cuml one makes fine sense, and these aren't benchmark notebooks so we don't have to worry too much about whether there is a slight overhead. So I think it works to keep the code as you have it.